### DISCUSSION TITLE
Use github_repo() only to retrieve files, not building anything?

### DISCUSSION THREAD

**Discussion by ringods** [thought-machine/please#2471] [Q&A]
Hello,

This is my first attempt on using Please. I aim to reuse files from a remote Github repository, so I can reuse them in later steps. I just tried with this in my `BUILD` file:

```python
github_repo(name="packer-macOS", repo="trodemaster/packer-macOS-11", revision="5f65ac7")
```

Executing Please on this, gives me this error:

```sh
$ plz build ///packer-macOS
Build stopped after 100ms. 1 target failed:
    ///packer-macOS//:packer-macOS
Can't build ///packer-macOS//:packer-macOS; there's no BUILD or BUILD.plz file in plz-out/subrepos/packer-macOS/
```

Indeed, there is no `BUILD` file there, nor do I want one there. How can I tell Please to not try to build it?

Side question: can I configure where the retrieved files are placed? I would prefer to "clone" the repo at `<project root>/packer-macOS/`.

**Comment by Tatskaari** [comment]
Labels that start with `///` index into a subrepo. They take the format `///repo_rule//some:target` where `//:repo_rule` is a rule that outputs a Please repo, and `//some:target` is a target within that repo. 

If you just want to download the github repo, use the standard syntax i.e. `plz build //:package-macOS`, or `deps = ["//:package-macOS"]`
